lib/dialects/postgres/data-types.js

```diff
@@ -80,6 +80,11 @@ BLOB.prototype.toSql = function() {
   return 'BYTEA';
 };
 
+var GEOMETRY = function() {
```

Comment:
There is no reason to have any special code in postgres - having all the code in abstract datatypes is fine